app/views/commodities/_commodity.html.erb

```diff
@@ -16,14 +16,16 @@
       <div class="description open" id="commodity-<%= commodity.short_code %>"><%= commodity.to_s.html_safe %></div>
 
       <div class="commodity__info">
-        <div class="vat" aria-describedby="commodity-vat-title">
-          <% vats = commodity.overview_measures.vat %>
-          <% if vats.count > 0 %>
-            <%= vats.map { |m| m.duty_expression.amount.to_i.to_s + "%" }.join(" or ").html_safe %>
-          <% else %>
-            &nbsp;
-          <% end %>
-        </div>
+        <% if TradeTariffFrontend::ServiceChooser.uk? %>
+          <div class="vat" aria-describedby="commodity-vat-title">
+            <% vats = commodity.overview_measures.vat %>
+            <% if vats.count > 0 %>
+              <%= vats.map { |m| m.duty_expression.amount.to_i.to_s + "%" }.join(" or ").html_safe %>
```

Comment:
Appreciate this is old code but wherever possible `.html_safe` is best avoided, this could use safe_join instead?

```ruby
<%= safe_join vats.map { |m| m.duty_expression.amount.to_i.to_s + "%" }, ' or ' %>
```